core/src/main/java/org/apache/accumulo/core/util/Halt.java

```diff
@@ -51,7 +51,9 @@ public static void halt(final int status, Runnable runnable) {
 
     try {
       // give ourselves a little time to try and do something
-      Threads.createThread("Halt Thread", () -> {
+      // TODO KEVIN RATHBUN doesn't matter if this is critical or not, halt() will be called in
+      // this method no matter what
+      Threads.createNonCriticalThread("Halt Thread", () -> {
```

Comment:
doesn't matter if this is critical or not, halt() will be called in this method no matter what